lib/render/FlowRenderer.cpp

```diff
@@ -29,8 +30,8 @@ FlowRenderer::FlowRenderer( const ParamsMgr*    pm,
                       FlowRenderer::GetClassType(),
                       instName,
                       dataMgr ),
-            _velocityField     ( 8 ),
-            _colorField        ( 2 ),
+            _velocityField     ( 9 ),
```

Comment:
How were these numbers determined? Please document.

Reply:
more document is added.